Products/Zuul/routers/zep.py

```diff
@@ -626,8 +630,8 @@ def _buildRequestFilters(self, uid, params, evids, excludeIds):
             log.debug('Found specific event ids, adding to params.')
             includeUuids = evids
 
-        includeFilter = self._buildFilter([uid], params, specificEventUuids=includeUuids)
         exclude_params = self._filterParser.findExclusionParams(params)
+        includeFilter = self._buildFilter([uid], params, specificEventUuids=includeUuids)
```

Comment:
`findExclusionParams()` will find and remove from `params` any search fields containing "!!", so it must be run before `_buildFilter()`